build-system/compile/post-closure-babel.js

```diff
@@ -86,8 +86,8 @@ exports.postClosureBabel = function () {
       file.contents,
       file.sourceMap
     );
-    const {compressed, terserMap} = terserMinify(code);
 
+    const {compressed, terserMap} = await terserMinify(code);
```

Comment:
Note that this won't propagate an error from terser into the pipeline. We need an explicit try/catch (since we're using async/await now), with the catch block doing `next(error)`

Reply:
Yes, that's been noted and will be addressed in a future PR.

https://github.com/ampproject/amphtml/pull/29705#discussion_r466734305